lib/rdkafka/abstract_handle.rb

```diff
@@ -45,7 +54,7 @@ def pending?
     # on the operation. In this case it is possible to call wait again.
     #
     # @param max_wait_timeout [Numeric, nil] Amount of time to wait before timing out.
-    #   If this is nil it does not time out.
+    #   If this is nil we will use WAIT_TIMEOUT value
```

Comment:
"If this is nil, will wait forever" would be more explicit IMHO. WAIT_TIMEOUT is an implementation detail